actionview/lib/action_view/helpers/tags/check_box.rb

```diff
@@ -38,6 +28,23 @@ def render
           end
         end
 
+        def attributes
+          options = options_with_hidden_attribute
+          options.delete("include_hidden")
```

Comment:
Calls to `Hash#delete` are destructive and are not idempotent. That means that the first call to `#attributes` will return a different value from subsequent calls.